app/containers/Avatar/index.tsx

```diff
@@ -9,96 +9,87 @@ import { IApplicationState, TSubscriptionModel, TUserModel } from '../../definit
 import Avatar from './Avatar';
 import { IAvatar } from './interfaces';
 
-class AvatarContainer extends React.Component<IAvatar, any> {
-	private subscription?: Subscription;
+const AvatarContainer = ({
+	server,
+	style,
+	text = '',
+	avatar,
+	emoji,
+	size,
+	borderRadius,
+	type,
+	children,
+	user,
+	onPress,
+	getCustomEmoji,
+	isStatic,
+	rid,
+	blockUnauthenticatedAccess,
+	serverVersion,
+	externalProviderUrl
+}: IAvatar): React.ReactElement => {
+	const [avatarETag, setAvatarETag] = useState<string | undefined>('');
+	let subscription: Subscription;
```

Comment:
Function components work different than class components.
This value was `this.subscription` was `bound` to the class, so it would keep its value through re-renders.

Function components are going to trigger the whole `body` on every render, so using `let` is going to lost previous data.
You're looking for either `useRef` or a custom hook.